precompiles/xcm/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "pallet-evm-precompile-xcm"
+authors = [ "Stake Technologies <devops@stake.co.jp>" ]
+description = "Basic XCM support for EVM."
+edition = "2021"
+version = "0.1.0"
+
+[dependencies]
+precompile-utils = { path = "../utils", default-features = false }
+pallet-evm-precompile-assets-erc20 = { path = "../assets-erc20", default-features = false }
+pallet-xcm = { path = "../../frame/pallet-xcm", default-features = false }
+num_enum = { version = "0.5.3", default-features = false }
+log = "0.4.16"
+
+# Substrate
+codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = [ "max-encoded-len" ] }
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+pallet-assets = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.19", default-features = false }
+
+# Frontier
+fp-evm = { git = "https://github.com/AstarNetwork/frontier", branch = "polkadot-v0.9.19", default-features = false }
+pallet-evm = { git = "https://github.com/AstarNetwork/frontier", branch = "polkadot-v0.9.19", default-features = false }
+
+# Polkadot
+xcm = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.19", default-features = false }
```

Comment:
Should we use our own fork here?

Not for this PR, but perhaps it would be good to rename our `pallet-xcm` since now it has the exact same name as the one used by polkadot/kusama/etc. runtimes.

Reply:
This is not pallet-xcm, it's XCM primitives. And yes, pallet-xcm referenced as `path` above.

Reply:
Omg, sorry, my bad, not sure what I saw :sweat_smile: